sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -1,10 +1,10 @@
 # Azure Form Recognizer client library for Python
 
-Azure Form Recognizer is a cloud service that uses machine learning to analyze text and structured data from your documents. It includes the following main features:
+Azure Document Intelligence (previously Form Recognizer) is a cloud service that uses machine learning to analyze text and structured data from your documents. It includes the following main features:
```

Comment:
is there a link or announcement somewhere that can be linked out?

Reply:
Good idea, I'll ping the service team to see if they have something I havent seen one since I've been looking through their docs, here's the tracking issue: https://github.com/Azure/azure-sdk-for-python/issues/31481